src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -43,6 +43,17 @@ class Vet360ProfileField extends React.Component {
     transactionRequest: PropTypes.object,
   };
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.transaction && !this.props.transaction) {
```

Comment:
Basically if the transaction is finished, we want to set the focus back to the edit link associated with this profile field

Reply:
This is the same logic we use elsewhere in the app for managing focus.

Reply:
would this helper be useful? https://github.com/department-of-veterans-affairs/vets-website/blob/29a2269f1cd98db9a8946a5a956f69c1442eb811/src/platform/utilities/ui/index.js#L17